python/pyspark/ml/base.py

```diff
@@ -309,7 +309,8 @@ def setPredictionCol(self, value):
         """
         return self._set(predictionCol=value)
 
-    @abstractproperty
+    @property  # type: ignore
```

Comment:
These ignores are not strictly required now, but I think it is OK to keep them, as there is not real harm here and they're likely to prove required in the future.